src/user/index.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+import { Table } from 'user/table'
+
+export const User = () => <Table />
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/125454334-fe24ae79-77b9-4850-a307-6a12c5edda4d.png)
Was this `table` directory really necessary, since we only export the `Table` component?

Reply:
Done